packages/cli/src/commands/apps/stacks/index.ts

```diff
@@ -0,0 +1,35 @@
+import {ux} from '@oclif/core'
+import {Command, flags} from '@heroku-cli/command'
+import * as _ from 'lodash'
+import color from '@heroku-cli/color'
+import {Stacks} from '../../../lib/types/stacks'
+import {App} from '../../../lib/types/app'
+
+export default class StacksIndex extends Command {
+  static description = 'show the list of available stacks'
+  static topic = 'apps'
+  static aliases = ['stack']
+
+  static flags = {
+    app: flags.app({required: true}),
+  }
+
+  async run() {
+    const {flags} = await this.parse(StacksIndex)
+
+    const {body: app} = await this.heroku.get<App>(`/apps/${flags.app}`)
+    let {body: stacks} = await this.heroku.get<Stacks>('/stacks')
```

Comment:
I would keep the original `Promise.all` for these requests. No reason to do them in series.

Reply:
Linting was yelling at me since only app was the only actual const variable. What would you recommend here?

Reply:
Make them `const` and you can then rename the sorted stacks below.
```
const sortedStacks = _.sortBy(stacks, 'name')
ux.styledHeader(`${color.app(app.name)} Available Stacks`)
for (const stack of sortedStacks) {
```